passes/opt/wreduce.cc

```diff
@@ -529,6 +529,42 @@ struct WreducePass : public Pass {
 						module->connect(sig, Const(0, GetSize(sig)));
 					}
 				}
+
```

Comment:
This isn't a peepopt!

Reply:
No, but it simplifies pattern matching involving `$div/$mod/$pow` cells if we do at least a bit of wreduction on those cells as well.